FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -202,18 +204,25 @@ - (BOOL)isApplicationPreWarmed {
     return NO;
   }
 
+  BOOL isPrewarmed = NO;
+
   NSDictionary<NSString *, NSString *> *environment = [NSProcessInfo processInfo].environment;
   BOOL activePrewarmFlagValue = [environment[@"ActivePrewarm"] boolValue];
-  if ([self isActivePrewarmEnabled] && activePrewarmFlagValue == YES) {
-    return YES;
+  if (activePrewarmFlagValue == YES) {
+    isPrewarmed = isPrewarmed || [self isActivePrewarmEnabled];
+    [self.activeTrace incrementMetric:kFPRAppCounterNameActivePrewarm byInt:1];
+  } else {
+    [self.activeTrace incrementMetric:kFPRAppCounterNameActivePrewarm byInt:0];
   }
 
-  if ([self isDoubleDispatchEnabled] &&
-      [doubleDispatchTime compare:applicationDidFinishLaunchTime] == NSOrderedAscending) {
-    return YES;
+  if ([doubleDispatchTime compare:applicationDidFinishLaunchTime] == NSOrderedAscending) {
```

Comment:
If devs disable double dispatch, doubleDispatchTime is always nil. Can you confirm`[doubleDispatchTime compare:applicationDidFinishLaunchTime] == NSOrderedAscending` always return false?

Reply:
In that case this expression will return nil, so this if-block is skipped. Then line 221 will be checked, if doubleDispatchTime is null that block will also be skipped, so in the end no custom metric will be added to the trace if dev disables double dispatch